packages/components/ng-ovh-payment-method/src/components/integration/component/adyen/index.js

```diff
@@ -0,0 +1,25 @@
+import angular from 'angular';
+import 'angular-translate';
+import '@ovh-ux/ng-translate-async-loader';
+
+import '@ovh-ux/ui-kit';
+import './index.less';
+
+import component from './component';
+import service from './service';
+import ADYEN_CONSTANTS from './constants';
+
+const moduleName = 'ngOvhPaymentMethodIntegrationComponentAdyen';
+
+angular
+  .module(moduleName, [
+    'ngTranslateAsyncLoader',
+    'pascalprecht.translate',
+    'oui',
+  ])
+  .component(component.name, component)
```

Comment:
IMO we should use a string instead of `component.name`, not easy to find where a component is declared.